File: Metode avansate de Programare/subiecte/ro/practic/rezolvari/26.01.2016 ro/Luca Corneliu/src/main/java/client1/service/ServiceException.java
package client1.service;

/**
 * Created by luca_ on 26.01.2016 at 14:30.
 */
public class ServiceException extends RuntimeException {
    public ServiceException(Exception e) {
        super(e);
    }

    public ServiceException(String message) {
        super(message);
    }
}

File: Metode avansate de Programare/Teme/familydoctor/Saptamana 8-14/tcp client-server/familydoctor/FamilyDoctorCommon/Service/ServiceException.cs
using System;
using System.Runtime.Serialization;

namespace FamilyDoctor.Service
{
    [Serializable]
    public class ServiceException : Exception
    {
        public ServiceException()
        {
            
        }

        public ServiceException(string message)
            : base(message)
        {
            
        }

        public ServiceException(string message, Exception innerException)
            : base(message, innerException)
        {
            
        }

        protected ServiceException(SerializationInfo info, StreamingContext context)
            : base(info, context)
        {
            
        }
    }
}